package-lock.json

```diff
@@ -72,6 +72,20 @@
         "typescript": "4.4.2"
       }
     },
+    ".yalc/@dasch-swiss/dsp-js": {
+      "version": "7.1.0+3be3775d",
+      "extraneous": true,
+      "license": "AGPL-3.0",
+      "dependencies": {
+        "@babel/helper-compilation-targets": "^7.16.7",
+        "@types/jsonld": "^1.5.6",
+        "json2typescript": "1.4.1",
+        "jsonld": "^5.2.0"
+      },
+      "peerDependencies": {
+        "rxjs": "6.x"
+      }
+    },
```

Comment:
the yalc package shouldn't be here anymore

Reply:
hmm, I'll have a look. I thought removing the yalc package and adding the real package would be enough to get rid of it in the lock file.

Reply:
I removed it manually in [fdb4118](https://github.com/dasch-swiss/dsp-app/pull/733/commits/fdb411808a93ab616d71ec4e3e7539ed73d81704)